contentcuration/contentcuration/assets/channelList/views/Channel/ChannelModal.vue

```diff
@@ -0,0 +1,705 @@
+<template>
+  <VDialog
+    v-model="open"
+    attach="body"
+    fullscreen
+  >
+    <VCard>
+      <VToolbar card prominent color="blue">
+        <VBtn icon class="hidden-xs-only" @click="close">
+          <VIcon>clear</VIcon>
+        </VBtn>
+        <VToolbarTitle>
+          {{ channel.name }}
+        </VToolbarTitle>
+      </VToolbar>
+      <VLayout row wrap>
+        <VFlex xs12 sm12 md3>
+          <ThumbnailUpload v-model="thumbnail" :readonly="!canEdit" />
+        </VFlex>
+        <VFlex xs12 sm12 md8>
+          <VAutocomplete
```

Comment:
Sorry, forgot to add this to my review! Couple notes here:
- There is a [LanguageDropdown](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/static/js/edit_channel/sharedComponents/LanguageDropdown.vue) available in develop now, but feel free to update it to handle v-model like you have it here (right now, it's just used under the publish modal)
- I've been meaning to do this, but could you please make this field required?